adapter/redisquota/redisquota.go

```diff
@@ -80,7 +80,7 @@ func (builder) ValidateConfig(cfg adapter.Config) (ce *adapter.ConfigErrors) {
 	return
 }
 
-func (builder) NewQuotasAspect(env adapter.Env, c adapter.Config, d map[string]*adapter.QuotaDefinition) (adapter.QuotasAspect, error) {
+func (builder) NewQuotasAspect(env adapter.Env, c adapter.Config, _ map[string]*adapter.QuotaDefinition) (adapter.QuotasAspect, error) {
```

Comment:
@geeknoid BTW today none of the quota adapters use the map of quota definitions - should we remove that parameter?